src/components/file-upload/FileUpload.tsx

```diff
@@ -20,8 +20,8 @@ export interface FileUploadProps {
   error?: boolean;
 
   /** Display file info if uploaded file property has data */
-  uploadedFile?: UploadedFileProps;
-  onRemove: () => void;
+  uploadedFile?: UploadedFileProps | null;
+  onRemove?: (() => void) | null;
```

Comment:
What do you think about using `null` or `undefined`? `null` is appropriate for props that the Developer would normally expect to use. `undefined` is best for completely optional props that don't affect the functionality of the component.
```suggestion
  uploadedFile?: UploadedFileProps;
  onRemove?: (() => void);
```

Reply:
I just made up my mind. Thanks for the suggestion